### PR TITLE
Claim AnkiDroid as our team project

### PR DESCRIPTION
**Pull request by Danny7226**
Claim AnkiDroid as our team project

### CODE REVIEW

**Comment by SWE-265P** [comment]
Just to verify, are you proposing [this repo](https://github.com/ankidroid/Anki-Android) for your project?

File: projects.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+|RuntimeTerrors |  OpenRefine  |       500k             | Familiarity with the languages used and adequate documentation  |                                                      
+|            |                 |                        |                                                         |
+>>>>>>> 0a44a438968d48bc8f53dab239199de4074382cb
+|            |                 |                        |                                                         |
+|            |                 |                        |                                                         |
+|            |                 |                        |                                                         |
+|            |                 |                        |                                                         |
+|            |                 |                        |                                                         |
+|            |                 |                        |                                                         |
+=======
```
Comment (SWE-265P) [inline projects.md]: Please resolve the merge conflict here.

**Comment by Danny7226** [comment]
> Just to verify, are you proposing [this repo](https://github.com/ankidroid/Anki-Android) for your project?

Yes!

File: projects.md (hunk 1 above)
Comment (SWE-265P) [reply projects.md]: @Danny7226 please wait with resolving until you actually have resolved the merge conflict here. :)

Comment (Danny7226) [reply projects.md]: Done! Thank you!

**Comment by SWE-265P** [comment]
After discussion we decided that Ankidroid is not approved. This project contains only around 50k loc. SVG and XML are ultimately both markup languages. As a result the number of lines of code is not enough to be considered for a project for this course. 
image

Here are some suitable alternatives:
- [Apache Druid](https://github.com/apache/druid)
- [jmonkeyengine](https://github.com/jMonkeyEngine/jmonkeyengine)
- [jenkins](https://github.com/jenkinsci/jenkins)

I will close this PR for now.

**Pull request closed without merging**